loader/CMakeLists.txt

```diff
@@ -104,6 +104,7 @@ set(NORMAL_LOADER_SRCS
     gpa_helper.c
     loader.c
     log.c
+    settings.c
```

Comment:
Technically it's good practice to also include the settings.h in the CMake as well for VS / XCode users.